charts/careers/templates/secrets.yaml

```diff
@@ -0,0 +1,55 @@
+{{- if .Values.externalSecrets.enabled -}}
+apiVersion: kubernetes-client.io/v1
+kind: ExternalSecret
+metadata:
+  name: {{ include "careers.fullname" . }}
+spec:
+  backendType: secretsManager
+  dataFrom:
+  - {{ .Values.externalSecrets.secretName }}
+---
+apiVersion: kubernetes-client.io/v1
+kind: ExternalSecret
+metadata:
+  name: {{ include "careers.fullname" . }}-hook
+  annotations:
+    "helm.sh/hook": pre-install,pre-upgrade
+    "helm.sh/hook-weight": "-10"
+    "helm.sh/hook-delete-policy": before-hook-creation,hook-succeeded
+spec:
+  backendType: secretsManager
+  dataFrom:
+  - {{ .Values.externalSecrets.secretName }}
+{{- else }}
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ include "careers.fullname" . }}
+data:
```

Comment:
Do we want the secrets in here base64 encoded? I see it's not the production value just curious about this technique

Reply:
I was ok with it, like you said these aren't real values, but committing integration/functional test secrets to source code is weird for sure, but I personally like that pattern.